docs: improve contributing.md and some minor improvements (#41)
invia-flights/blitzly: docs/CONTRIBUTING.md

6. So no code is repeated we use [`update_figure_layout`](https://invia-flights.github.io/blitzly/utils/#blitzly.etc.utils.update_figure_layout) to update the layout of the figure.

7. [`save_show_return`](https://invia-flights.github.io/blitzly/utils/#blitzly.etc.utils.save_show_return) is used to save the figure as an HTML file - if needed, shows the figure, and returns the figure. The use of this function is also mandatory.

8. When you work on the implementation try to make the implementation as generic as possible. In other words, try to wrap a powerful Plotly implementation in your blitzly function and pass as many arguments as possible to the user while giving default arguments. In addition to this please also provide a `plotly_kwargs` argument which allows the user to pass additional keyword arguments to the Plotly function. This way the user can customize the plot as much as possible.

9. Don't forget to write some meaningful unit tests. The tests should cover all possible use cases of the function and should be located in the `tests/test_cases` folder and should be named `test_<function_name_with_use_case>.py`.

> 10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.me`](https://github.com/invia-flights/blitzly#available-plots-so-far-) file. If you think your plot implementation is worth to be documented even more, feel free to add it together with a short description to the end of the [`playground.ipynb`](https://github.com/invia-flights/blitzly/blob/main/examples/playground.ipynb) notebook.

Reviewer: ```suggestion
10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.md`]
```

> 10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.me`](https://github.com/invia-flights/blitzly#available-plots-so-far-) file. If you think your plot implementation is worth to be documented even more, feel free to add it together with a short description to the end of the [`playground.ipynb`](https://github.com/invia-flights/blitzly/blob/main/examples/playground.ipynb) notebook.

Reviewer: ```suggestion
10. Documentation: Please add your new blitly ⚡️ plot to the list in the [`README.md`](https://github.com/invia-flights/blitzly#available-plots-so-far-) file. If you think your plot implementation is worth to be documented even more, feel free to add it together with a short description to the end of the [`playground.ipynb`](https://github.com/invia-flights/blitzly/blob/main/examples/playground.ipynb) notebook.
```

> That's it! 🎉 Happy ploding! 📊👩‍💻 If you have any questions feel free to contact us!

Reviewer: ```suggestion
That's it! 🎉 Happy plotting! 📊👩‍💻 If you have any questions feel free to contact us!
```

Author: That was actually on purpose. It's supposed to be a mash-up of plotting and coding.
